pkg/cmd/pr/create/create.go

```diff
@@ -287,11 +289,15 @@ func createRun(opts *CreateOptions) error {
 			return fmt.Errorf("could not compute title or body defaults: %w", defaultsErr)
 		}
 	} else if opts.Autofill {
-		if defaultsErr != nil {
+		if defaultsErr != nil && !(titleProvided || bodyProvided) {
 			return fmt.Errorf("could not compute title or body defaults: %w", defaultsErr)
 		}
-		title = defs.Title
-		body = defs.Body
+		if !titleProvided {
+			title = defs.Title
+		}
+		if !bodyProvided {
+			body = defs.Body
+		}
```

Comment:
since we already set title and body to `opts.Title` and `opts.Body` respectively, I added these conditions to prevent overwriting them with defaults.